m4_sessions/1708-inning/group_six.md

```diff
@@ -1,9 +1,54 @@
 # Group Six
 
-### Session Title:
+### Session Title: The not so subtle art of not giving a F#^k
 
 ### Session Outcomes: 
 
+Give students the tools to determine for themselves what is important. 
+Worksheet with top five priorities and top five things you don't give a fuck about for personal and professional life
+
+#### Topic Ideas
+* Remove and filter out external pressures and learn to recognize when they affecting your decision making
+
+* The perspective of privilage. Being self aware. Getting worked up over little things compared to real levels of adversity and how it relates to the way we code. Without minifying the personal struggle.
+
+* Not falling into social confitioning traps and understanding the hedonic adaptation and how it affects your hapiness
+
+* Stop wasting your time so you can use it better and feel more fulfilled.
+
 ### Session Outline:
 
+Ice Breaker
+
+Hand out worksheet for priority list and explain
+
+TedTalk
+
+Turn and Talk 2-3 about discussion points
+
+Group talk about those specific discussion points
+
+Topic 1 - Intro
+Topic 1 Group Discussion
+
+Topic 2 - Intro
+Topic 2 Group Discussion
+
+game time
+
+Topic 3 -intro
+Topic 3 Groupd Discussion
+
+Conclusion and Sharing Worksheets(if comfortable)
```

Comment:
Will you come together to close whole group?  How can you review all of the information so that people walk away feeling clear about next steps or what they might change to live more fulfilled?